sys-apps/plocate/files/plocate-1.1.9-no-systemd-check.patch

```diff
@@ -0,0 +1,38 @@
+From 0b1e568bc66df9fdb0bfd6f54b4ff2e540769dbe Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Arsen=20Arsenovi=C4=87?= <arsen@aarsen.me>
+Date: Thu, 19 Aug 2021 02:36:54 +0200
+Subject: [PATCH] unconditionally configure systemd service
+
+---
+ meson.build | 15 +++------------
+ 1 file changed, 3 insertions(+), 12 deletions(-)
+
+diff --git a/meson.build b/meson.build
+index dc14eb1..9a3b3fe 100644
+--- a/meson.build
++++ b/meson.build
+@@ -76,18 +76,9 @@ updatedb_conf_man = configure_file(input: 'updatedb.conf.5.in',
+                configuration: conf_data)
+ install_man(updatedb_conf_man)
+ 
+-if get_option('install_systemd')
```

Comment:
Why is this patch necessary, given that `install_systemd` defaults to true and the ebuild not modifying the meson option? Is it because `systemunitdir` may be not autodetected if sytemd is not installed? If so, why not set this variable using systemd.eclass' `systemd_get_systemunitdir`, which also "works" if systemd is not installed? I think we could then also drop the `systemd_dounit` in the ebuild's `src_install` (but I have to check first if there is a gentoo policy that `systemd_dounit` has to be used, or not).

Reply:
The systemd dep was unconditionally checked before, it can be removed now.